src/providers/Comscore.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Comscore
+ * https://direct.comscore.com/clients/help/FAQ.aspx#faqTagging
+ *
+ * @class
+ * @extends BaseProvider
+ */
+
+class ComscoreProvider extends BaseProvider {
+  constructor() {
+    super();
+    this._key = "COMSCORE";
+    this._pattern = /sb\.scorecardresearch\.com.+(?<!\.js)$/;
```

Comment:
We'll need to refactor this. Firefox doesn't support lookbehinds yet :(
https://caniuse.com/#feat=js-regexp-lookbehind

Reply:
What I am excluding here is their js library which loads alongside pixel calls sometimes. It has a similar URL structure (sb.scorecardresearch.com...), but ends in ".js" How about this?

`/sb\.scorecardresearch\.com.+[^.js)]$/`

Reply:
Perfect, thanks!

Reply:
Quick update to this that I just noticed..
`/sb\.scorecardresearch\.com.+[^.js)]$/`
would fail when the URL ended in a `.`, `j`, or `s`

How about this instead?
`sb\.scorecardresearch\.com(?!.*\.js($|[?#]))`

Reply:
Done!